src/xrp/xrp_helpers.c

```diff
@@ -78,17 +78,20 @@ static size_t xrp_encode_base58_address(const base58_buf_t *in, xrp_address_t *o
     return outlen;
 }
 
-void xrp_public_key_hash160(xrp_pubkey_t *pubkey, uint8_t *out) {
+cx_err_t xrp_public_key_hash160(xrp_pubkey_t *pubkey, uint8_t *out) {
     union {
         cx_sha256_t shasha;
         cx_ripemd160_t riprip;
     } u;
     uint8_t buffer[32];
+    cx_err_t error = CX_INTERNAL_ERROR;
 
     cx_sha256_init(&u.shasha);
-    cx_hash_no_throw(&u.shasha.header, CX_LAST, pubkey->buf, sizeof(pubkey->buf), buffer, 32);
+    error =
+        cx_hash_no_throw(&u.shasha.header, CX_LAST, pubkey->buf, sizeof(pubkey->buf), buffer, 32);
+    if (error != CX_OK) return error;
```

Comment:
Not really a fan of single line if, and if the linter is applied it will be weird
If you really want to return the cx_error you don't really have an elegant way out but if the caller only needs to know success/error, you can simplify
```
if (cx_hash_no_throw(&u.shasha.header, CX_LAST, pubkey->buf, sizeof(pubkey->buf), buffer, 32) != CX_OK) {
    return -1;
};
```

Reply:
I am not fan of the single line as well, but yes, it is requested by the linter...
I can update the code, removing the `error` variable and use a `if` statement directly if you want.